calcom/cal.com
# feat: Plain chat

## What does this PR do?

This PR replaces the main intercom chat with plain.com's live chat feature.

## Mandatory Tasks (DO NOT REMOVE)

- [Y ] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [ Y] I have updated the developer docs in /docs if this PR makes changes that would require a [documentation change](https://cal.com/docs). If N/A, write N/A here and check the checkbox.
- [ Y] I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?



- Are there environment variables that should be set?
     PLAIN_CHAT_ID=
     PLAIN_CHAT_HMAC_SECRET_KEY=
- What are the minimal test data to have?
     Post requests to plain.com
- What is expected (happy path) to have (input and output)?
     Able to see user messages in plain
- Any other important info that could help to test that PR

## Checklist
